pkg/reconcilermanager/controllers/controller.go

```diff
@@ -23,8 +23,12 @@ import (
 // SetupWithManager
 type Controller interface {
 	reconcile.Reconciler
-	// SetupWithManager registers the controller with the controller-manager
-	SetupWithManager(mgr controllerruntime.Manager, watchFleetMembership bool) error
+	// Setup error handlers, indexers, and transformers with controller-manager.
```

Comment:
This doesn't seem to be the case anymore. We can delete the comment.
Also, if `Setup` doesn't do real work, do we still need the interface?

Reply:
Removed Setup. I initially moved the indexers there, but later realized they required the CRDs. So I had to remove the indexes all together.